ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1139,10 +1156,122 @@ KdbpContextFromPrevTss(
     *pTss = Tss;
     Context->Eip = Eip;
     Context->Ebp = Ebp;
+#endif
     return TRUE;
 }
 #endif
 
+#ifdef _M_AMD64
+
+typedef struct _RUNTIME_FUNCTION *PRUNTIME_FUNCTION;
+typedef struct _UNWIND_HISTORY_TABLE *PUNWIND_HISTORY_TABLE;
+PRUNTIME_FUNCTION NTAPI RtlLookupFunctionEntry(ULONG_PTR,DWORD64*, PUNWIND_HISTORY_TABLE);
+
+typedef struct _KNONVOLATILE_CONTEXT_POINTERS *PKNONVOLATILE_CONTEXT_POINTERS;
+#define UNW_FLAG_NHANDLER 0
+
+PEXCEPTION_ROUTINE
+NTAPI
+RtlVirtualUnwind(
+    _In_ ULONG HandlerType,
+    _In_ ULONG64 ImageBase,
+    _In_ ULONG64 ControlPc,
+    _In_ PRUNTIME_FUNCTION FunctionEntry,
+    _Inout_ PCONTEXT Context,
+    _Outptr_ PVOID *HandlerData,
+    _Out_ PULONG64 EstablisherFrame,
+    _Inout_opt_ PKNONVOLATILE_CONTEXT_POINTERS ContextPointers);
+
+static
+BOOLEAN
+GetNextFrame(
```

Comment:
Note to self: This dumps more frames than are actually available, creating an excessive backtrace. Should be investigated, but it's good enough for now.